Embedded Java Engine Samples/Basic/Data Source Usage/BasicOData/BasicOData/src/main/java/BasicOData.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2012 by Windward Studios, Inc. All rights reserved.
+ *
+ * This program can be copied or used in any manner desired.
+ */
+
+import java.awt.*;
+import java.io.*;
+
+import net.windward.datasource.DataSourceProvider;
+import net.windward.datasource.odata.ODataDataSource;
+import net.windward.xmlreport.ProcessPdf;
+import net.windward.xmlreport.ProcessReport;
+import net.windward.xmlreport.ProcessReportAPI;
+import net.windward.env.SystemWrapper;
+
+/*
+ * A sample usage of the Windward Java Engine. This is the first building block to a great report!
+ *
+ * This sample takes in a datasource and template file--in this case, data.xml and template.docx from
+ * the data directory--and produces the report, stored as report.pdf in the out directory.
+ *
+ * Take a look, the results are amazing!
+ */
+public class BasicOData {
```

Comment:
This sample should have placeholder data for connection strings and templates since this doesn't work anymore. I.e. {{YOUR_CONNECTION_STRING}}